### ISSUE TITLE
Check CKV_AZURE_35 doesnt detect that rule is set in azurerm_storage_account_network_rules

### ISSUE THREAD

**Issue by JamesWoolfenden** [bridgecrewio/checkov#475]
**Describe the bug**
This rule checks to see if you have -to "Ensure default network access rule for Storage Accounts is set to deny", you can do this in the storage account resource or as a separate resource - azurerm_storage_account_network_rules. I  you use the later checkov says you havent set to deny when you have.

**To Reproduce**
Create a storage account with the rule but add something like:
```
resource "azurerm_storage_account_network_rules" "store" {
  resource_group_name  = var.resource_group.name
  storage_account_name = azurerm_storage_account.store.name

  default_action = var.default_action
}


variable "default_action" {
  type        = string
  default     = "Deny"
}

```
Run checkov and itll tell you you didn't set to deny and raise CKV_AZURE_35: "Ensure default network access rule for Storage Accounts is set to deny" when you did.

**Expected behavior**
should be valid to set as separate resource to deny.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: win10
 - Checkov Version 1.0.426

**Comment by dsfrederic** [comment]
Please reopen this. I'm experiencing the same issue @JamesWoolfenden @schosterbarak

**Comment by dsfrederic** [comment]
And actually the same is happening for CKV_AZURE_35

**Comment by dsfrederic** [comment]
In my CI CD tool it's complaining about the azurerm_storage_account resource but not the network_rules

![image](https://user-images.githubusercontent.com/28486949/158377950-354728bf-214b-44d2-ad9e-b30a4e2f519f.png)
